src/cli/commands/install.js

```diff
@@ -232,7 +230,7 @@ export class Install {
       Object.assign(this.resolutions, json.resolutions);
       Object.assign(manifest, json);
 
-      const pushDeps = (depType, {hint, visibility, optional}) => {
+      const pushDeps = (depType, {hint, optional}, isUsed) => {
```

Comment:
I've changed the callsites so there's just one use of `true` now. Eventually if we add a `--dev-only` flag we'd have to remove it anyway so I don't think there's much value.